fluent-react/example/src/App.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode, ReactElement } from "react";
+import { Localized } from "@fluent/react";
+import { FluentDateTime } from "@fluent/bundle";
+import { Hello } from "./Hello";
+import { LocalizedSignIn } from "./SignIn";
+
+export function App() {
+    let date = new Date();
```

Comment:
Hmm, wouldn't this break the `It's HH:MM` widget, though? I agree that it's a bit weird of a use-case. Perhaps it would make sense to change the example string to `It was HH:MM when you first opened this page?` 🤔

Reply:
Or change it to displaying the current month?

Reply:
If you do want to show the current time, it'd require some useEffect-ing and setInterval-ing. Given the complexity that adds, I'd also vote for sth like "It was X when you first opened this page".

Reply:
I went for the current weekday in the end. Thanks!